sound/hda/intel-dsp-config.c

```diff
@@ -543,6 +543,15 @@ static const struct config_entry config_table[] = {
 		.device = PCI_DEVICE_ID_INTEL_HDA_LNL_P,
 	},
 #endif
+
+	/* Panther Lake */
+#if IS_ENABLED(CONFIG_SND_SOC_SOF_PANTHERLAKE)
+	{
+		.flags = FLAG_SOF | FLAG_SOF_ONLY_IF_DMIC_OR_SOUNDWIRE,
+		.device = PCI_DEVICE_ID_INTEL_HDA_PTL,
+	},
+#endif
```

Comment:
At this point the SOF stack will be probad (if DMIC or SDW) but SOF support fro PTL is coming only in the next two patches...
Is it still a nitpick to ask for a cleaner order of patches?

Reply:
The order doesn't matter for intel-dsp-config:
a) the patch will be picked by Takashi, not Mark Brown
b) the intel-dsp-config stuff is called by the HDA and SOF drivers, not the other way around. So if the SOF support is lacking the code is not used just yet.

Reply:
Put differently, the patches will be sent in four batches.
a) the PCI ID. Takashi will typically pick it up and provide a branch for Mark
b) the snd-hda-intel changes
c) the asoc changes
d) the snd-intel-dspconfig

the fact that the patches are all listed in the same PR does not mean we will send them in the same patchset when upstreaming.